kubernetes/linux/Dockerfile.multiarch

```diff
@@ -113,16 +113,17 @@ COPY --from=builder /lib/libjq.so.1 /lib/libonig.so.5 /lib/
 FROM distroless_image AS vulnscan
 COPY .trivyignore .trivyignore
 RUN ["/bin/bash", "-c", "curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.39.0"]
-RUN trivy rootfs --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM --skip-files "/usr/local/bin/trivy" /
-RUN trivy rootfs --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM /usr/lib
-RUN trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM --skip-files "/usr/local/bin/trivy" / > /dev/null 2>&1 && trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM /usr/lib > /dev/null 2>&1
+RUN ["/bin/bash", "-c", "trivy rootfs --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM --skip-files \"/usr/local/bin/trivy\" /"]
+RUN ["/bin/bash", "-c", "trivy rootfs --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM /usr/lib"]
+RUN ["/bin/bash", "-c", "trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM --skip-files \"/usr/local/bin/trivy\" / > /dev/null 2>&1 && trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM /usr/lib > /dev/null 2>&1"]
 
 # Revert to base layer before vulnscan
 FROM distroless_image AS ContainerInsights
 # force the trivy stage to run
 # docker buildx (BUILDKIT) does not build stages which do not affect the final stage
 # by copying over a file we create a dependency
 # see: https://github.com/docker/build-push-action/issues/377
+COPY --from=vulnscan /usr/local/bin/trivy /usr/local/bin/trivy

```

Comment:
@jatakiajanvi12 , i am seeing trivy detecting vulns without this change. in which scenario, trivy was not running?